cookbooks/bach_hive/recipes/config.rb

```diff
@@ -0,0 +1,15 @@
+%w(hive webhcat hcat hive-hcatalog).each do |w|
+  directory "/etc/#{w}/conf.#{node.chef_environment}" do
+    owner 'root'
+    group 'root'
+    mode 00755
+    action :create
+    recursive true
+  end
+
+  bach_hive_alternatives "update-#{w}-conf-alternatives" do
```

Comment:
Should we move alternatives to `Library` cookbook? This will be used by multiple cookbooks/components.